src/template/style/base_custom_color.ts

```diff
@@ -0,0 +1,26 @@
+import { BaseTemplate } from '../base/base';
+
+export class BaseCustomColor extends BaseTemplate {
+    private _dartString: string;
+
+    constructor(fileName: string, suffix: string) {
+        super(fileName, suffix);
+        this._dartString = `import 'package:flutter/material.dart';
+
```

Comment:
this template have to be empty, because any custom color is different

Reply:
fixed